docs/src/test-advanced.md

```diff
@@ -200,6 +201,41 @@ export const test = base.extend<{ saveLogs: void }>({
 });
 ```
 
+## Launching a development web server during the tests
+
+To launch a web server during the tests, use the `webServer` option in the [configuration file](#configuration-object).
+
+Playwright Test does automatically detect if a localhost URL like `http://localhost:3000` gets printed to the stdout.
+The port from the printed URL gets then used to check when its accepting requests and passed over to Playwright as a
+`baseURL`. You can also manually specify a `port` or additional environment variables, see [here](#configuration-object).
+
+```js js-flavor=ts
+// playwright.config.ts
+import { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'npm run start',
+    timeout: 120 * 1000,
+  },
+};
+
+export default config;
+```
+
+```js js-flavor=js
+// @ts-check
+/** @type {import('@playwright/test').PlaywrightTestConfig} */
+const config = {
+  webServer: {
+    command: 'npm run start',
+    timeout: 120 * 1000,
+  },
+};
+
+mode.exports = config;
+```
+
```

Comment:
Please add a test example here that navigates to a relative path making use of baseURL.